### PR TITLE
Update stats.md

### PR DESCRIPTION
**Pull request by LouderthanThunder**
Fleshing out stats and what they mean

### CODE REVIEW

**Comment by Lambosaurus** [comment]
I pushed a commit merging this with some of my content. A fair bit got rewritten and moved in the process.

File: systems/stats.md [hunk 1]
```diff
@@ -1,8 +1,51 @@
 
-## Defence
+# Attributes
+These attributes intrinsic to your character. They represent the base capabilities of your body and mind. These will only change rarely throughout a characters journey.
```
Comment (Hoppelite) [inline systems/stats.md:3]: Missing 'are'

File: systems/stats.md [hunk 2]
```diff
@@ -1,8 +1,51 @@
 
-## Defence
+# Attributes
+These attributes intrinsic to your character. They represent the base capabilities of your body and mind. These will only change rarely throughout a characters journey.
+
+These are all recorded as a numeric modifier. An attribute of `0` represents an unexceptional value that a non adventuring human citizen might have.
```
Comment (Hoppelite) [inline systems/stats.md:5]: Specifying human in a potentially non human world. Maybe just person or character

File: systems/stats.md [hunk 3]
```diff
@@ -1,8 +1,51 @@
...
+## Intellect
+The ability to problem solve, and accurately percieve the world around them.
+Intelligence represents a characters knowledge and wisdom into a particular subject and will be used for inspection tests, investigation tests and as a primary stat for arcane magic
 
-Defence is a `number`
+
+Intellect is primarially used for [arcane magic](spells.md#Arcane-magic) 
+
+## Will
+A character with high will . They are resistant to [spells](spells.md)
```
Comment (Hoppelite) [inline systems/stats.md:34]: Something seems to be missing here

**Review by Hoppelite — CHANGES_REQUESTED** [review]
Just a couple touch ups

File: systems/stats.md (hunk 3 above)
Comment (Lambosaurus) [reply systems/stats.md:34]: I forgot to delete that line

**Pull request merged**